olive/model/config/model_config.py

```diff
@@ -88,6 +90,8 @@ class ModelConfig(ConfigBase):
 
     type: str
     config: dict
+    # For enable_fast_mode only. Otherwise, it will be None.
+    loaded_model: Any = None
```

Comment:
This is a compromise solution. Since we decided to return model config for pass run, we should somehow pass loaded model back to engine. Another option is changing returning model config back to model handler, where I tried in [another branch](https://github.com/microsoft/Olive/blob/xiaoyu/lmim/olive/engine/engine.py#L997). But that will break a bunch of current logics and tests.

Reply:
we could allow system accept OliveModelHandler or ModelConfig as input and output for the fast mode.